src/kartograf/filters/element_change.py

```diff
@@ -49,3 +49,21 @@ def filter_element_changes(
         filtered_mapping[i] = j
 
     return filtered_mapping
+
+
+def filter_hybridization_changes(
+    molA: Chem.Mol, molB: Chem.Mol, mapping: dict[int, int]
+) -> dict[int, int]:
+    """Forces a mapping to exclude any alchemical hybridization changes."""
```

Comment:
can we add another quick line giving examples of what it would filter, e.g. your `-CC >> -C=C` test in words here